src/Components/Auth/auth.js

```diff
@@ -11,13 +11,11 @@ import { useState } from "react";
 
 export const AuthCheck = () => {
     const [username,setusername]=useState('')
-    const [pass,setpass]=useState('')
-    const [err,seterr]=useState(false)
-    const navigate = useNavigate();
+      const [pass,setpass]=useState('')
+      const [err,seterr]=useState(false)
+      const navigate = useNavigate();
 
      function Authtry() {
```

Comment:
If you are using ES6 everywhere then use it here also. Other than that everything is looking great, awesome work.